Formula/o/opentap.rb

```diff
@@ -0,0 +1,81 @@
+class Opentap < Formula
+  desc "Fast and easy development and execution of automated tests"
+  homepage "https://opentap.io"
+  url "https://github.com/opentap/opentap/archive/refs/tags/v9.24.2.tar.gz"
+  sha256 "a8015e1781899690027e2c65c0452e9dc91914d71c824bc1c2c6985a034aa55a"
+  license "MPL-2.0"
+
+  depends_on "dotnet"
+
+  uses_from_macos "unzip" => :build
+
+  def install
+    os = OS.mac? ? "MacOS" : "Linux"
+    arch = Hardware::CPU.intel? ? "x64" : "arm64"
+    package_output_path = buildpath/"OpenTAP.TapPackage"
+    dotnet = Formula["dotnet"]
+    args = %W[
+      --configuration Release
+      /p:AssemblyVersion=#{version}
+      /p:FileVersion=#{version}
+      /p:InformationalVersion=#{version}
+      /p:Version=#{version}
+    ]
+
+    # Build OpenTAP
+    system "dotnet", "build", "tap/tap.csproj", *args
+    system "dotnet", "build", "Cli/Tap.Cli.csproj", *args
+    system "dotnet", "build", "Package/Tap.Package.csproj", *args
+    system "dotnet", "build", "BasicSteps/Tap.Plugins.BasicSteps.csproj", *args
+
+    # Replace the version in the package.xml file
+    inreplace "package.xml", "$(GitVersion)", version.to_s
+
+    # Create the TapPackage
+    Dir.chdir("bin/Release") do
+      with_env(Platform: os, Architecture: arch, Sign: "false") do
+        system "./tap.sh", "package", "create", "../../package.xml", "-o", package_output_path
+      end
+    end
+
+    # Extract the TapPackage
+    mkdir_p buildpath/"output"
+    system "unzip", package_output_path, "-d", buildpath/"output"
+
+    # Install the output of the TapPackage
+    prefix.install Dir[buildpath/"output/*"]
+
+    # Even though the LC_ID_DYLIB path is an @rpath type, brew still updates it to an absolute path.
+    # This means we need to update the libgit2 hash in the OpenTAP package.xml file, since the hash has changed.
+    # This only needed on MacOS
+    # Use command `otool -L libgit2-b7bad55.dylib.arm64` to get path.
+    # Read more about LC_ID_DYLIB here: https://forums.developer.apple.com/forums/thread/736719 and https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/DynamicLibraries/100-Articles/RunpathDependentLibraries.html
```

Comment:
Does it compile it's own libgit?

Reply:
No, it uses the one shipped with [Libgit2Sharp](https://www.nuget.org/packages/LibGit2Sharp). Libgit2Sharp does compile their own libgit for various platforms: https://github.com/libgit2/libgit2sharp.nativebinaries

Reply:
It should really be using the homebrew package for libgit. 

https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae